go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	google.golang.org/grpc v1.33.2
 	google.golang.org/protobuf v1.26.0
 	k8s.io/api v0.20.4
+	k8s.io/apiextensions-apiserver v0.20.2 // indirect
```

Comment:
@ajanth97 does this still show up in the `go.mod` after you run this command:

```shell
go mod tidy
```

?

Reply:
Yes I removed the mage binary and added it to my PATH. And yes the go.mod file still shows those depenedencies